testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -5846,6 +5847,19 @@ QUnit.module('single line mode', {
 
         $inputWrapper.trigger('mousedown');
     });
+
+    QUnit.test('mousedown should not be prevented when input field clicked (T1046705)', function(assert) {
+        assert.expect(1);
```

Comment:
Why should ew use `expect`? Is there really some async code execution?

Reply:
Nope 
Just copypasted previus test whithout check as always, sry